joomla/joomla-cms
# [4.3][Guided Tours] Cleanup Steps model

Pull Request for Issue # .

### Summary of Changes
This PR makes several clean up to Steps model:

- Remove getTable method. It is not used for a List model
- Do not need to set filter.search and filter.published states, it is handled by parent class.
- Correct getStoreId method, it needs to contains filter.tour_id state value
- Rename variable from $tour_id to $tourId to follow camel case naming convension


### Testing Instructions

1. Install Joomla base on download package generated by this PR to make sure you have guided tours available in the installation. Or if you have a site with guide tours already, apply this PR
2. Access to Guided Tours, then view steps of a tour and make sure nothing is broken on that screen.

### Actual result BEFORE applying this Pull Request
Contains unnecessary code


### Expected result AFTER applying this Pull Request
Unnecessary code removed


### Link to documentations
Please select:
